CMakeSettings.json

```diff
@@ -1,126 +1,157 @@
+            "generator": "Ninja",
+            "configurationType": "Debug",
+            "inheritEnvironments": [ "msvc_x64_x64" ],
+            "buildRoot": "${projectDir}\\out\\vs2019\\${name}",
+            "installRoot": "${projectDir}\\out\\vs2019\\${name}\\install",
+            "cmakeCommandArgs": "",
+            "buildCommandArgs": "",
+            "ctestCommandArgs": "",
+            "variables": [
+                {
+                    "name": "WITH_STL",
+                    "value": "True",
+                    "type": "BOOL"
+                },
+                {
+                    "name": "WITH_OTLP",
+                    "value": "True",
+                    "type": "BOOL"
+                },
+                {
+                    "name": "WITH_EXAMPLES",
+                    "value": "true",
+                    "type": "BOOL"
+                },
+                {
+                    "name": "WITH_PROMETHEUS",
+                    "value": "True",
+                    "type": "BOOL"
+                }
+            ]
         },
         {
-          "name": "WITH_EXAMPLES",
-          "value": "true",
-          "type": "BOOL"
+            "name": "stdlib-x64-Release",
+            "generator": "Ninja",
+            "configurationType": "Release",
+            "inheritEnvironments": [ "msvc_x64_x64" ],
+            "buildRoot": "${projectDir}\\out\\vs2019\\${name}",
+            "installRoot": "${projectDir}\\out\\vs2019\\${name}\\install",
+            "cmakeCommandArgs": "",
+            "buildCommandArgs": "",
+            "ctestCommandArgs": "",
+            "cmakeToolchain": "",
+            "variables": [
+                {
+                    "name": "WITH_STL",
+                    "value": "True",
+                    "type": "BOOL"
+                },
+                {
+                    "name": "WITH_OTLP",
+                    "value": "True",
+                    "type": "BOOL"
+                },
+                {
+                    "name": "WITH_EXAMPLES",
+                    "value": "true",
+                    "type": "BOOL"
+                },
+                {
+                    "name": "WITH_PROMETHEUS",
+                    "value": "True",
+                    "type": "BOOL"
+                }
+            ]
         },
         {
-          "name": "WITH_PROMETHEUS",
-          "value": "True",
-          "type": "BOOL"
+            "name": "stdlib-x64-Release-contrib",
+            "generator": "Ninja",
+            "configurationType": "Release",
+            "buildRoot": "${projectDir}\\out\\vs2019\\${name}",
+            "installRoot": "${projectDir}\\out\\vs2019\\${name}\\install",
+            "cmakeCommandArgs": "",
+            "buildCommandArgs": "",
+            "ctestCommandArgs": "",
+            "cmakeToolchain": "",
+            "inheritEnvironments": [ "msvc_x64_x64" ],
+            "variables": [
+                {
+                    "name": "WITH_CONTRIB",
```

Comment:
This is funny format that Visual Studio 2019 uses for `CMake` build combos. This combo, for illustrative purposes, is built with all possible exporters and `WITH_CONTRIB`.

Reply:
The other second option is to specify the path explicitly:

```json
        {
            // These environment variables apply to all configurations.
            "environments": [
                {
                    "OPENTELEMETRY_CONTRIB_PATH": "${projectDir}\\..\\opentelemetry-cpp-contrib"
                }
            ],
            "name": "stdlib-x64-Release-contrib"
        }
```

Then a local custom snapshot can be used, e.g. my checked out tree of contrib side-by-side with the main is built in one build when I build the main, with all artifacts and tests ran at the same time.